go/test/endtoend/messaging/message_test.go

```diff
@@ -129,15 +134,16 @@ func TestMessage(t *testing.T) {
 	require.NoError(t, err)
 	qr = exec(t, conn, "select time_next, epoch from vitess_message where id = 1")
 	next, epoch = getTimeEpoch(qr)
+	jitter += maxJitter
 	// epoch could be 1 or 2, depending on how fast the row is updated
 	switch epoch {
 	case 1:
-		if !(start < next && next < start+3e9) {
-			t.Errorf("next: %d. must be about 1s after start: %d", next/1e9, start/1e9)
+		if !(start < next && next < (start+jitter)+3e9) {
+			t.Errorf("next: %d. must be about 1s after start: %d", next/1e9, (start+jitter)/1e9)
 		}
 	case 2:
-		if !(start+2e9 < next && next < start+6e9) {
-			t.Errorf("next: %d. must be about 3s after start: %d", next/1e9, start/1e9)
+		if !(start+2e9 < next && next < (start+jitter)+6e9) {
+			t.Errorf("next: %d. must be about 3s after start: %d", next/1e9, (start+jitter)/1e9)
```

Comment:
Jitter would have happened twice here, so maxJitter would be epoch * 1.333

Reply:
Thanks, made the change to use epoch*maxJitter. There will no jitter in Epoch 0: is that correct?

Reply:
> 	// have backoff be +/- 33%, whenever this is injected, append (:min_backoff, :jitter)
	jitteredBackoff := "FLOOR((%a<<ifnull(epoch, 0)) * %a)"

Correct, no jitter. By default, the jitter is also windowed around the min/max backoff. If there are no settings, minBackoff is the same as ackWaitTimeout and no limit on maxBackoff.